nimy_lisp.nimble

```diff
@@ -18,8 +18,9 @@ let
   genDocCmd = "nim doc " & srcFile
   htmlFile = "src" / (pkgName & ".html")
 
-task test, "Run tests via 'nim doc' and runnableExamples":
+task test, "Run tests via 'nim doc' and runnableExamples and tests in tests dir":
   exec(genDocCmd)
+  exec "nim c -r tests/tWhenLet.nim"
```

Comment:
Thanks! This is perfectly fine and actually great. I can see that all tests won't go well as runnableExamples.